py/dml/c_backend.py

```diff
@@ -3357,7 +3345,6 @@ def generate_cfile_body(device, footers, full_module, filename_prefix):
                 raise ICE(e.site, 'error during late compile stage')
             eom_linemark(func.method.rbrace_site)
             out('}\n\n', preindent = -1)
-        reset_line_directive()
```

Comment:
This does the wrong thing when `gather_size_statistics` is on. That should be solvable though.